test-requirements.txt

```diff
@@ -2,6 +2,7 @@
 coverage==4.5.2
 pep8==1.7.1
 flake8==3.7.7
+hacking
```

Comment:
Probably not a bad idea to pin it to a specific version to avoid surprises when a new version if published / similar. This has bitten us many times already in the past, same for ``flake8-copyright``.

Although checking ``hacking`` docs (https://pypi.org/project/hacking/) it looks like ``hacking`` itself installs specific versions of ``pep8``, etc. so there could be potential conflicts down the road.

Reply:
@Kami Pinned both flake8-copyright and hacking in test-requirements.txt.  Why did we pin a specific version of pep8 (pep8==1.7.1)?

Reply:
New versions (> 2.0.0, IIRC), only work with Python 3.